public/index.html

```diff
@@ -130,13 +133,98 @@ <h3 class="pl-1 fontHeading">Create New Task</h3>
                 </div>
             </div>
             <div class="pt-2 pb-5">
-                <button type="submit" class="btn btn-primary" id="submitCreateNewTask">Submit</button>
+                <button type="submit" class="btn btn-primary" id="submitCreateNewSprint">Submit</button>
```

Comment:
Why the button named createNewTask is being changed to CreateNewSprint... This will distort our whole codebase.